lib/streamlit/server/routes.py

```diff
@@ -54,6 +54,12 @@ def set_extra_headers(self, path):
             self.set_header("Cache-Control", "public")
 
 
+class AssetsFileHandler(tornado.web.StaticFileHandler):
+    def set_default_headers(self):
+        if allow_cross_origin_requests():
+            self.set_header("Access-Control-Allow-Origin", "*")
+
+
```

Comment:
Is this correct? I think we *always* want this header to be set, regardless of whether `allow_cross_origin_requests()` is true. (Otherwise, serving assets to components won't work unless the user disables CORS for all of Streamlit.)

Reply:
(Assuming I am correct about this, we should definitely also have a comment documenting why this is the case.)